bootstrap/011_warehouse_schedules.sql

```diff
@@ -1,18 +1,96 @@
 
 -- NB. Table is created in 090_post_setup.sql because you cannot call python procs from the setup.sql
 
-CREATE OR REPLACE PROCEDURE ADMIN.UPDATE_WH_SCHEDULES()
-    RETURNS text
-    language python
-    runtime_version = "3.10"
-    handler = 'run'
-    packages = ('snowflake-snowpark-python', 'pydantic')
-    imports = ('{{stage}}/python/crud.zip')
-    EXECUTE AS OWNER
+-- CREATE TABLE IF NOT EXISTS internal.task_warehouse_schedule(RUN TIMESTAMP_LTZ(9), SUCCESS BOOLEAN, OUTPUT VARIANT)
+
+CREATE OR REPLACE PROCEDURE INTERNAL.UPDATE_WAREHOUSE_SCHEDULES(last_run timestamp_ltz, this_run timestamp_ltz)
+    RETURNS VARIANT
+    language sql
  AS
-$$
-from crud import WarehouseSchedulesTask
-def run(session):
-    task = WarehouseSchedulesTask(session)
-    return task.run()
-$$;
+DECLARE
+    task_outcome variant default (select object_construct());
+BEGIN
+    -- The task calls this procedure with NULL and lets the procedure figure out the details.
```

Comment:
i wonder if this might be neater using a session parameter?

Reply:
Rather than passing in `NULL`, you mean? I didn't think we could do that in an owner's rights procedure.

Reply:
that null is hella ugly so we have to think of something to do.